kinetic/distribution.yaml

```diff
@@ -2434,6 +2434,11 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/dynamixel-workbench-msgs.git
       version: kinetic-devel
     status: developed
+  dynamixel_control_hw:
+    doc:
+      type: git
+      url: https://github.com/resibots/dynamixel_control_hw.git
+      version: kinetic-devel
```

Comment:
There does not appear to be a kinetic-devel branch, I'd guess that you want to index master?

```suggestion
      version: master
```

Reply:
Yes exactly, thank you !

Reply:
Please accept my suggestion or make the same change yourself to this branch and then we can merge this.